lib/bundles/inspec-compliance/api.rb

```diff
@@ -238,5 +243,11 @@ def self.server_version_from_config(config)
       return nil unless config['version'].is_a?(Hash)
       config['version']['version']
     end
+
+    def self.determine_server_type(url, insecure)
+      return 'automate' if Compliance::HTTP.get(url + '/compliance/version', nil, insecure).code == '401'
```

Comment:
It's more idiomatic to return symbols for something like this rather than strings.

`return :automate if ...`

Reply:
I always prefer going the idiomatic route. Fixed. Thanks @adamleff!